WebService/WebService/Base Classes/Bar.cs

```diff
@@ -19,7 +19,7 @@ public class Bar
         public string BarLocation { get; set; }
 
         //A list of drinks available at this bar along with the prices
-        [NotMapped]
+        //[NotMapped]
```

Comment:
could just delete this line since if we mapped it once, we probably won't make it not mapped again